packages/xrpl/test/integration/requests/submit.ts

```diff
@@ -74,4 +76,25 @@ describe('submit', function () {
 
     assert.deepEqual(submitResponse, expectedResponse)
   })
+
+  // Reproduces the 'txnNotFound' case
+  it('submit & quickly check status without waiting', async function () {
```

Comment:
Does this actually replicate the situation? It seems that it's not even using the new code that you wrote.

Reply:
removed.